_posts/2017-08-24-monthly-checkin.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: "Monthly Check-in"
+slug: monthly-checkin
+text: 'We will have a check-in on the status of Toronto Mesh working groups.'
+location: Pauper's Pub, 539 Bloor St W
+locationLink: http://www.openstreetmap.org/node/303251580
+date: 2017-08-24
+startTime: '19:00'
+endTime: '21:00'
+---
+
+Join us at {{page.startTime | date:"%-I:%M%P "}} on {{ page.date | date: '%B %d, %Y' }} for our monthly check-in with Toronto Mesh working groups. This month we will be meeting at [{{ page.location }}]({{ page.locationLink }}).  
+Join the {% include icon-matrix.html %} chat for further updates!
+
+A recap of items discussed will be available in our [{{ page.date | date: '%B %d, %Y' }} check-in notes](https://github.com/tomeshnet/documents/blob/master/meeting_notes/{{ page.date | date: "%Y%m%d" }}_check-in.md).
```

Comment:
Do you want to have this here? (only needed for notes) I thought we had a conditional to only show this block after events... once meeting notes were up